phpunit.xml

```diff
@@ -32,4 +32,7 @@
         <log type="coverage-text" target="./build/coverage.txt" showUncoveredFiles="true"/>
         <log type="coverage-clover" target="./build/logs/clover.xml" showUncoveredFiles="true"/>
     </logging>
+    <php>
```

Comment:
I removed this on purpose, since i don't want to have these versions hardcoded on the code anymore, i just haven't added it to Travis yet.